x-pack/platform/plugins/shared/alerting_v2/server/rule_executor/constants.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * Fixed data stream name for ES|QL alert events written by `alerting_v2`.
+ */
+export const ALERT_EVENTS_INDEX = '.alerts-events';
```

Comment:
Up for discussion, no strong opinions..

```suggestion
export const ALERT_EVENTS_INDEX = '.alerts_events';
```